config/checker-framework-suppressions/checker-nullness-optional-interning-suppressions.xml

```diff
@@ -2743,6 +2743,20 @@
     </details>
   </checkerFrameworkError>
 
+  <checkerFrameworkError unstable="false">
+    <fileName>src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java</fileName>
+    <specifier>initialization.field.uninitialized</specifier>
+    <message>the default constructor does not initialize field ignoreAnnotationShortNames</message>
+    <lineContent>private Set&lt;String&gt; ignoreAnnotationShortNames;</lineContent>
+  </checkerFrameworkError>
+
+  <checkerFrameworkError unstable="false">
+    <fileName>src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java</fileName>
+    <specifier>initialization.field.uninitialized</specifier>
+    <message>the default constructor does not initialize field immutableClassShortNames</message>
+    <lineContent>private Set&lt;String&gt; immutableClassShortNames;</lineContent>
+  </checkerFrameworkError>
```

Comment:
should we disable this Rule is Checker ?

I see we do this suppression again and again, and there are ~100  of such suppression already.

Reply:
See my previous comment at https://github.com/checkstyle/checkstyle/pull/12639#discussion_r1070312108

This only conflicts with pitest.

Reply:
yes, so that is why I am questioning role of rule in our project, looks like very conflicting with java Checks that has initialization methods `beginTree` and `beginProcessing`.

Reply:
I am fine to disable, but this makes me question if there are any rules from this process that we do like.

Reply:
We will prove by practice that some rules unfortunately does not match us or conflicting with tools that we like more.
For now we suppressing. If we endup in suppressing of all eventually, it would be good to show/share publicly this experience.

Reply:
Let’s make an issue for disablement if this rule and discuss there.